app/assets/stylesheets/messages/base.css.scss

```diff
@@ -0,0 +1,19 @@
+.message {
+  text-align: left;
+  margin-top: 1em;
+  margin-left: 3em;
+  border-bottom: 1px solid transparentize(black, 0.9);
+
+
+  .message-header {
+    display: inline;
+
+    .h {
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 3